content/en/docs/collector/getting-started.md

```diff
@@ -1,358 +1,84 @@
-tar -xvf otelcol_{{% param collectorVersion %}}_darwin_amd64.tar.gz
-```
-
-{{% /tab %}} {{% tab ARM %}}
-
-```sh
-curl --proto '=https' --tlsv1.2 -fOL https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_darwin_arm64.tar.gz
-tar -xvf otelcol_{{% param collectorVersion %}}_darwin_arm64.tar.gz
-```
-
-{{% /tab %}} {{< /tabpane >}}
-
-Every Collector release includes an `otelcol` executable that you can run after
-unpacking.
-
-## Windows Packaging
-
-Windows [releases][] are packaged as gzipped tarballs (`.tar.gz`) and will need
-to be unpacked with a tool that supports this compression format.
-
-Every Collector release includes an `otelcol.exe` executable that you can run
-after unpacking.
-
-## Local
-
-Builds the latest version of the collector based on the local operating system,
-runs the binary with all receivers enabled and exports all the data it receives
-locally to a file. Data is sent to the container and the container scrapes its
-own Prometheus metrics. The following example uses two terminal windows to
-better illustrate the collector. In the first terminal window run the following:
-
-```sh
-git clone https://github.com/open-telemetry/opentelemetry-collector.git
-cd opentelemetry-collector
-make install-tools
-make otelcorecol
-./bin/otelcorecol_* --config ./examples/local/otel-config.yaml
-```
-
-In a second terminal window, you can test the newly built collector by doing the
-following:
-
-```sh
-git clone https://github.com/open-telemetry/opentelemetry-collector-contrib.git
-cd opentelemetry-collector-contrib/examples/demo/server
-go build -o main main.go; ./main & pid1="$!"
-cd ../client
-go build -o main main.go; ./main
-```
-
-To stop the client, use type <kbd>Ctrl-C</kbd>. To stop the server, use the
-`kill $pid1` command. To stop the collector, type <kbd>Ctrl-C</kbd> in its
-terminal window as well.
-
-**Note:** The commands shown above demonstrate the process in a bash shell.
-These commands may vary slightly for other shells.
-
-[data collection]: /docs/concepts/components/#collector
-[deployment methods]: ../deployment/
-[readme.md]:
-  https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/examples/demo
-[opentelemetry helm charts]: /docs/kubernetes/helm/
-[opentelemetry operator]: /docs/kubernetes/operator/
-[getting started with opentelemetry on hashicorp nomad]:
-  https://github.com/hashicorp/nomad-open-telemetry-getting-started
-[releases]:
-  https://github.com/open-telemetry/opentelemetry-collector-releases/releases
+- Learn about the different modes of the Collector in
+  [Deployment Methods](../deployment/).
+- Familiarize yourself with the Collector
+  [configuration](/docs/collector/configuration) files and structure.
+- Try additional components using the
+  [opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib)
+  project.
```

Comment:
```suggestion
- Explore available components in the [registry](https://opentelemetry.io/ecosystem/registry/?language=collector)
- Learn how to [build a custom collector with the OpenTelemetry Collector Builder (OCB)](/docs/collector/custom-collector/)
```

This is an opportunity to promote the registry, I think in that context it is also good to share the existence of OCB, since we want to discourage people from using otel-collector-contrib distro and build their own with the components they need.